app/jenkins-for-jira-ui/src/components/ServerManagement/ServerManagement.test.tsx

```diff
@@ -1,14 +1,171 @@
 import React from 'react';
```

Comment:
Some tests in here a new but others have been yanked out of ConnectionPanel coz I had to hoist up some of the requests so made more sense to test from the parent component.